redisson/src/main/java/org/redisson/redisnode/RedisNode.java

```diff
@@ -286,7 +289,25 @@ public void clusterFlushSlots() {
 
     @Override
     public RFuture<Map<ClusterSlotRange, Set<String>>> clusterSlotsAsync() {
```

Comment:
I would recommend change this interface to Map<ClusterSlotRange, List<NodeInfo>>, where NodeInfo contains metadata for the node, like port, address, etc.